### PR TITLE
Manage magic link

### PR DESCRIPTION
**Pull request by huynguyennovem**
- Story: #1023 
- Demo 1 (Invitation screen UI):

https://user-images.githubusercontent.com/29337364/139430852-96c55f00-e7fa-4529-bc18-020fa6ca060b.mp4

### CODE REVIEW

**Comment by huynguyennovem** [comment]
- Demo 2 (Invitation email functionality):

https://user-images.githubusercontent.com/29337364/139691088-e814c834-a940-478d-aa48-32096dbf931b.mp4

**Comment by huynguyennovem** [comment]
- Demo 3 (Invitation functionality):

https://user-images.githubusercontent.com/29337364/139827929-5f072c9f-5a0c-4569-bc43-0be6689b082f.mov

**Comment by tlle14** [comment]
> * Demo 3 (Invitation functionality):
> 
>  Screen.Recording.2021-11-02.at.5.08.47.PM.mov

Hi @huynguyennovem in Demo3, it's missing the popup confirm that the current link will be expired
Please refer #1023 , it said:
`When I click on "generate a new link", I have a confirmation popup saying the previous won't be usable anymore. If I press yes, a new link will be generated.`

**Review by EvGen94 — APPROVED** [review]

**Review by EvGen94 — APPROVED** [review]

**Comment by huynguyennovem** [comment]
- Demo 
    - generate link API
    - send email invitation API

https://user-images.githubusercontent.com/29337364/141757276-6eaa026a-52fe-40af-bdb7-2dc75016d4b8.mp4

File: lib/pages/magic_link/invitation_people_page.dart [hunk 1]
```diff
@@ -0,0 +1,346 @@
...
+  _handleClickOnButtonInviteByEmail() {
+    NavigatorService.instance.navigateToInvitationPeopleEmail(_urlNotifier.value);
+  }
+
+  _handleClickOnLink() {
+    Clipboard.setData(new ClipboardData(text: _urlNotifier.value)).then((_){
+      ScaffoldMessenger.of(context).showSnackBar(
+        SnackBar(
+          backgroundColor: Colors.white,
+          margin: EdgeInsets.fromLTRB(10.0, 20.0, 10.0, MediaQuery.of(context).size.height - 110),
```
Comment (EvGen94) [inline lib/pages/magic_link/invitation_people_page.dart:324]: Consider using Dim instead of MediaQuery.of(context).size.height

Comment (huynguyennovem) [reply lib/pages/magic_link/invitation_people_page.dart:324]: Okay

**Pull request merged**